variables-and-memory/problem-set-variables-and-memory.md

```diff
@@ -16,20 +16,20 @@
 What is the value of `result` after the function `is_letter_in_word` is invoked as shown below?
 
 ```py
-1  def is_letter_in_word(word, letter):
-2       """
-3       input: word (string) and letter (single character)
-4       output: boolean
-5       """
-6       if letter not in word:
-7           result = False
-8       else: 
-9           result = True
-10
-11      return result
-12
-13 result = True
-14 is_letter_in_word("hello", "x"):
```

Comment:
The explanation refers to line numbers, so if we remove the line numbers, the explanation should be reworded  as well.